Makefile

```diff
@@ -296,6 +302,11 @@ test-e2e:
 	cd deploy; source ./../e2e/local-e2e.sh && run $(driver)
 	$(GO) test -v $(shell $(GO) list ./e2e)
 
+.PHONY: actions-e2e
+actions-e2e:
```

Comment:
Let's not create additional actions. We can parameterize and pass corresponding arguments in the actions.